www/js/components/add-point-hours.js

```diff
@@ -1,17 +1,22 @@
 import React, { Component } from 'react';
-import { CardText, Button } from 'react-mdl';
-
+import { CardText, Button, Icon } from 'react-mdl';
+import DropDown from './drop-down';
 // import redux components
 import { connect } from 'react-redux';
 
 import { addPointHours, removePointHours } from '../actions/new-point-actions';
 
+const weekDays = [
+  "Weekdays", "Weekends", "Monday", "Tuesday",
+  "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"
+];
+
 export class AddPointHours extends Component {
```

Comment:
It may be useful to turn the rows into simpler row components, and have this component contain a list of them. (future PR stuff)